addons/mail/static/src/models/message/message.js

```diff
@@ -821,8 +822,8 @@ function factory(dependencies) {
             ],
             inverse: 'messages',
         }),
-        tracking_value_ids: attr({
-            default: [],
+        trackingValues: one2many('mail.message_tracking_value', {
+            inverse: 'message',
```

Comment:
`isCausal: true` could be added here to ensure proper clean up (not really necessary for the code to work but always better)